test/Billing.Test/Controllers/FreshdeskControllerTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Text.Json;
+using Bit.Billing.Controllers;
+using Bit.Core.Entities;
+using Bit.Core.Repositories;
+using Bit.Core.Settings;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Billing.Test.Controllers
+{
+    public class FreshdeskControllerTests
+    {
+        private const string ApiKey = "TESTFRESHDESKAPIKEY";
+        private const string WebhookKey = "TEST_FRESHDESK_WEBHOOKKEY";
+        private const string TicketId = "TEST_FRESHDESK_TICKETID";
+        private const string Email = "TEST@EMAIL.COM";
+
+        private static (FreshdeskController, IUserRepository, IOrganizationRepository) CreateSut(
+            string freshdeskApiKey)
+        {
+            var userRepository = Substitute.For<IUserRepository>();
+            var organizationRepository = Substitute.For<IOrganizationRepository>();
+            var organizationUserRepository = Substitute.For<IOrganizationUserRepository>();
+
+            var billingSettings = Options.Create(new BillingSettings
+            {
+                FreshdeskApiKey = freshdeskApiKey,
+                FreshdeskWebhookKey = WebhookKey
+            });
+            var globalSettings = new GlobalSettings();
+            globalSettings.BaseServiceUri.Admin = "https://test.com";
+
+            var mockHttpMessageHandler = Substitute.ForPartsOf<MockHttpMessageHandler>();
+            var mockResponse = new HttpResponseMessage(System.Net.HttpStatusCode.OK);
```

Comment:
I don't think the response handler should be coded in the creation to always return the same expected value

Reply:
I coded it like this because the test only needs a mocked OK response for both requests